packages/circuit-ui/cli/migrate/icons-v2.ts

```diff
@@ -0,0 +1,309 @@
+/**
```

Comment:
Not really happy about this codemod, open to suggestions. I find it very verbose and hard to follow. But I think it also does the job 🤷 I'll test it locally on one of our apps before merging